src/common/components/userMatchModal/userMatchModal.component.spec.js

```diff
@@ -25,7 +25,7 @@ describe( 'userMatchModal', function () {
 
   describe( '$onInit', () => {
     beforeEach( () => {
-      spyOn( $ctrl.verificationService, 'getContacts' ).and.returnValue( Observable.of( [] ) );
+      spyOn( $ctrl.verificationService, 'getContacts' ).and.callFake( () => Observable.of( [] ) );
```

Comment:
What's the motivation for this switch to `callFake`? I've been slowly moving things to use `returnValue` cuz it doesn't need to be wrapped in a function and therefore is shorter. I didn't realize it would work when I first started writing these tests. It's trivial, I'm just curious.